cmd/api-errors.go

```diff
@@ -860,6 +861,11 @@ var errorCodes = errorCodeMap{
 		Description:    "Remote service connection error - please check remote service credentials and target bucket",
 		HTTPStatusCode: http.StatusNotFound,
 	},
+	ErrReplicationBandwidthLimitError: {
+		Code:           "XMinioAdminReplicationBandwidthLimitError",
+		Description:    "Bandwidth limit for remote target must be atleast 100Mbps",
```

Comment:
We should perhaps stick to 100MBytes instead of `100Mbits` since server always sees in bytes